command/peering/exportedservices/exported_services_test.go

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package exportedservices
```

Comment:
Can we add a case where a peering connection exists between clusters but there are no exported services to increase coverage?

Reply:
Also are we testing enterprise cases (output containing namespaces and partitions) in this?